purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -88,24 +87,20 @@ class Purchases internal constructor(
     val store: Store
         get() = purchasesOrchestrator.store
 
-    private val lifecycleHandler: AppLifecycleHandler by lazy {
-        AppLifecycleHandler(this)
-    }
-
-    init {
-        purchasesOrchestrator.dispatch {
-            // This needs to happen after the billing client listeners have been set. This is because
-            // we perform operations with the billing client in the lifecycle observer methods.
-            ProcessLifecycleOwner.get().lifecycle.addObserver(lifecycleHandler)
-        }
-    }
-
-    /** @suppress */
+    @Suppress("EmptyFunctionBlock")
+    @Deprecated(
+        "Will be removed in next major. Logic has been moved to PurchasesOrchestrator",
+        ReplaceWith("purchasesOrchestrator.onAppBackgrounded()"),
```

Comment:
Hmm I think we shouldn't add a `ReplaceWith` here... users won't be able to access `PurchaseOrchestrator`.

Reply:
I was getting a warning but maybe I can suppress the warning

Reply:
were these public before?

Reply:
Yeah, they were... So we are removing them on the next major